rabbitmq/receive.js

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env node
+
+const open = require("amqplib").connect("amqp://localhost");
+const WebSocket = require("ws");
+const _ = require("lodash");
+
+const wss = new WebSocket.Server({ port: 8080 });
+
+const queue = "transactions";
+const deliverables = [];
+const webSockets = [];
+
+console.log(" [*] Initializing RabbitMq ...");
+open
+  .then(async conn => {
+    const channel = await conn.createChannel();
+    const ok = await channel.assertQueue(queue, {
+      durable: false
+    });
+
+    if (ok) {
+      console.log(
+        " [*] RabbitMq initialized. Waiting for messages in %s. To exit press CTRL+C",
+        queue
+      );
+      channel.consume(
```

Comment:
I would move this entire logic into a new function:

```
const handleMsg = msg => {
   try {
            const transaction = JSON.parse(msg.content.toString());
            const deliverable = {
              ...transaction,
              tokens: transaction.size * 10
            };
   ....
}
```

It's always good to keep function as small as possible